wled00/set.cpp

```diff
@@ -264,7 +265,7 @@ void handleSettingsSet(AsyncWebServerRequest *request, byte subPage)
           #ifdef SOC_TOUCH_VERSION_2 // ESP32 S2 and S3 have a fucntion to check touch state but need to attach an interrupt to do so
```

Comment:
You will need to add logic to detect if previous button was a touch button and detach interrupt handler somewhere in this place.

Reply:
I took another look at this. 
the previous assignment of a pin does not need to be checked, the interrupt can just be detached (the function will actually just set the callback to a null pointer and set the threshold to max so the interrupt never triggers).
I see one issue with removing the 'detach interrupt' from the deallocate function: if a pin is changed from a touch input to a non-button pin (say an LED output), the interrupt should also be disabled. what would be a good place to do that?

Reply:
The same place attachment occurred IMO.